pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -854,6 +854,10 @@ type MgrSpec struct {
 	// +optional
 	// +nullable
 	Modules []Module `json:"modules,omitempty"`
+	// Whether host networking is enabled for CephManager. If not set, the network settings from CephCluster.spec.networking will be applied.
+	// +nullable
```

Comment:
Latest kubernetes guidelines recommend not using nullable, even for pointers.

Reply:
done